app/editor/src/features/content/list-view/utils/queryToFilter.ts

```diff
@@ -27,11 +27,15 @@ export const queryToFilter = (
       contentType: convertTo(search.contentType, 'string', filter.contentType),
       sourceId: convertTo(search.sourceId, 'number', filter.sourceId),
       otherSource: convertTo(search.otherSource, 'string', filter.otherSource),
-      productId: convertTo(search.productId, 'number', filter.productId),
       ownerId: convertTo(search.ownerId, 'number', filter.ownerId),
       userId: convertTo(search.userId, 'number', filter.userId),
       timeFrame: convertTo(search.timeFrame, 'number', filter.timeFrame),
-      productIds: filter.productIds,
+      productIds: search.productIds
+        ? search.productIds
+            .split('&')
+            .filter((item: string | string[]) => item.includes('productIds'))
+            .map((item: string) => Number(item.split('=')[1]))
```

Comment:
This will throw an error if everything isn't perfect (i.e. `productIds=1&productionIds=`).  Use `URLSearchParams.getAll()`